xaringan.Rproj

```diff
@@ -3,6 +3,7 @@ Version: 1.0
 RestoreWorkspace: Default
 SaveWorkspace: Default
 AlwaysSaveHistory: Default
+QuitChildProcessesOnExit: Default
```

Comment:
Which version of RStudio are you using? I have never seen such an option before. Usually I prefer not adding irrelevant changes to a PR if possible.